service/src/main/java/org/mskcc/smile/service/impl/TempoMessageHandlingServiceImpl.java

```diff
@@ -391,6 +393,12 @@ private void publishTempoSamplesToCBioPortal(Set<String> tumorPrimaryIds) throws
                     .setCmoSampleName(cmoSampleName)
                     .setAccessLevel(accessLevel)
                     .setCustodianInformation(custodianInformation)
+                    .setBaitSet(sampleMetadata.getBaitSet())
+                    .setGenePanel(sampleMetadata.getGenePanel())
+                    .setOncotreeCode(sampleMetadata.getOncotreeCode())
+                    .setCmoPatientId(sampleMetadata.getCmoPatientId())
+                    .setDmpPatientId(sampleService.getPatientAliasByTypeAndPrimaryId("dmpId", primaryId))
```

Comment:
recommending fetching patient from the database by the cmo patient id and then using that instance of Patient to get its dmp id